problems/5.loops.js

```diff
@@ -2,28 +2,43 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+function evenNumbers(num) {
+  let arr = []
+  for (let i = 0; i <= num; i += 2) {
+    arr.push(i)
+  }
+  return arr
 }
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
 * @param {number} num - The number to find multiples of
 * @returns {number[]} - An array of all multiples of {num} up to 1000
 */
-function multiplesToOneThousand() {
-
+function multiplesToOneThousand(num) {
+  let arr = []
+  for (let i = 0; i < 1000; i * num) {
```

Comment:
YOu're missing the reassignment in your step towards ending the condition. i += num. That's why you're getting an infiniate loop and hitting a memory error and causing you to fail nearly all the specs :-(